src/components/CalendarMonthGrid.jsx

```diff
@@ -33,6 +35,7 @@ const propTypes = forbidExtraProps({
   onMonthTransitionEnd: PropTypes.func,
   renderDay: PropTypes.func,
   transformValue: PropTypes.string,
+  daySize: PropTypes.number,
```

Comment:
yes, and ideally they'd be imported all from one place too - my personal preference is to export it from the leaf-est component that needs it.